website/docs/cloud-docs/workspaces/settings/drift-detection.mdx

```diff
@@ -0,0 +1,53 @@
+---
+page_title: Drift Detection - Workspaces - Terraform Cloud and Terraform Enterprise
+description: >-
+  Enable automatic jobs to check whether your real-world infrastructure matches your Terraform state file.
+---
+
+# Drift Detection
+
+-> **Note:** Drift detection is in beta. Please [provide feedback](https://forms.gle/BwjgvrKQcLbgqK3f7) about how we can improve. Drift detection is available in the paid **Business** tier. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
```

Comment:
these feel like 2 separate thoughts a little. maybe we should put in separate callouts? or just better combine them, like `Drift detection is available in the **Business** tier and currently in beta.....`

Reply:
ok i updated and I think it's a bit better!